File: AlarmApp/AlarmApp/ViewController.m
//
//  ViewController.m
//  AlarmApp
//
//  Created by 이상학 on 2015. 8. 18..
//  Copyright (c) 2015년 ryan. All rights reserved.
//

#import "ViewController.h"
#import "MainViewController.h"

@interface ViewController ()

@end

@implementation ViewController
@synthesize infoButton;
@synthesize mainViewController;

- (void)viewDidLoad {
    mainViewController *viewController = [self.storyboard instantiateViewControllerWithIdentifier:@"MainViewController"];
    mainViewController = viewController;
    [self.view insertSubview:viewController.view belowSubView:infoButton];
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a niåb.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: MapKit2/MapKit2/ViewController.swift
//
//  ViewController.swift
//  MapKit2
//
//  Created by 이상학 on 2015. 9. 25..
//  Copyright © 2015년 ryan. All rights reserved.
//

import UIKit
import CoreLocation
import MapKit


class ViewController: UIViewController {
    @IBOutlet weak var Map: MKMapView!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        var location = CLLocationCoordinate2DMake(42.68628, -73.85742)
        var span = MKCoordinateSpan (latitudeDelta: 0.02, longitudeDelta: 0.02) // 어느정도로 확대 할건지
        var region = MKCoordinateRegion(center: location, span : span)
        
        var annotation = MKPointAnnotation()
        annotation.coordinate = location
        annotation.title = "내 집 "
        annotation.subtitle = "비밀번호는 0000"
        
        Map.setRegion(region, animated: true)
        Map.addAnnotation(annotation)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

